scss/civicrm/administer/contribute/_contribute.scss

```diff
@@ -34,4 +34,15 @@
       margin-left: 20px;
     }
   }
+
+  .crm-contribution-page-configure-actions,
+  .crm-contribution-online-contribution-actions,
+  .crm-contribution-search-contribution-actions,
+  .crm-contribution-page-more {
+    .btn-slide {
```

Comment:
what's `btn-slide`, what issue is this style fixing?

Reply:
This is for **Manage contribution pages**, see first 2 screenshots

Reply:
ok why was the issue there in the first place?

Reply:
`.btn-slide` is class used globally to make dropdown of more button 
![image](https://user-images.githubusercontent.com/26058635/37084250-fd33f5ce-2217-11e8-8637-fb3a1c5c19f0.png)

On almost all pages (except 2 instance I found (another is manage events)) it works fine.
We need to make text visible here again so, user gets to know which buttons do what.

Reply:
In Vanilla, these buttons comes with text `more` which, we are hiding to make it like above,
![image](https://user-images.githubusercontent.com/26058635/37084864-a1ddeb06-2219-11e8-949b-ded7184b5f4a.png)
On other pages it gives clear indication that it's about more options but, on this specific page we need to show it back as there are more than one `.btn-slide` and one can not understand what is the meaning of this multiple 3 dots.